sensor_msgs/Image leftImage
sensor_msgs/Image rightImage
---
int64 distance
